guvnor-webapp/src/main/java/org/guvnor/client/perspectives/M2RepoPerspective.java

```diff
@@ -26,7 +26,8 @@
 import com.google.gwt.user.client.ui.PopupPanel;
```

Comment:
Yes this is true.. they will all need changing too. Are you to submit additional PRs?